code/tier_classification/README.md

```diff
@@ -59,3 +61,25 @@ If the exact variant is not present but the AA position matches with hotspot AA
 To see whether indel hotspot were present, the `key_clinical_findings_output` and `all_findings_output` were first filtered to contain only `Frame_Shift_Del`, `Frame_Shift_Ins`m `In_Frame_Del` and `In_Frame_Ins`. 
 For every entry in the indel hotspot file, the gene symbol and `HGVSp_Short` were used to query the `key_clinical_findings_output` and `all_findings_output` table to see whether they are present - if yes, then `Cancer Hotspot` annotation will be added to the `Variant_Properties` column.
 
+This module then add CIVIC curated annotation to the `all_findings_output` table using the script `civic_tier_anno.R` with the following rules:
+
+1. Annotate mutations
+1) First annotate specific SNV using `c_dot_civic.tsv` and `snv_remaining_civic.tsv`
+2) And then match by domain or exon and annotate any mutation within the domain/exon if they are in the CIVIC domain or exon files `mutation_exon_civic_annotated.tsv` and `mutation_domain_civic_annotated.tsv`
+3) insertions or deletions that are within exons described by CIVIC will be annotated as well (files used are `del_exon_civic_annotated.tsv` and `ins_exon_civic_annotated.tsv`)
+4) lastly, any remaining entries with mutations in genes listed in `mutation_general_civic.tsv` will be annotated 
+
+2. Annotate fusions using `fusion_column_only.tsv` and `fusion_word_only.tsv`
+1) `fusion_column_only.tsv` is used to annotate specific fusions with both gene partners
+2) then if any fusion has a fusion partner in the file `fusion_word_only.tsv`, the entry will be annotated
+
+3. Annotate expression results using `expression_all_civic.tsv`
+4. Annotate gain and loss from CNV results using `amplification_only_civic.tsv`, `cnv_loss_civic.tsv` and `deletion_general_civic.tsv`
+
+5. Annotate all the rest using `alteration_civic.tsv`
+For any of the entries that were not annotated by the previous 4 steps, they would be annotated if the gene is annotated in `alteration_civic.tsv`.
+
+Based on the civic evidence level, add additional tier or upgrade tier based on the following rules 
+1. Tier I: designated as A or B in `evidence_level_civic`
```

Comment:
similar question here - are you adding on civic to the current tier classification criteria (which is the goal)?

Reply:
Correct.